OrbitCore/Capture.cpp

```diff
@@ -237,14 +239,14 @@ void Capture::RegisterZoneName(uint64_t a_ID, const char* a_Name) {
 }
 
 //-----------------------------------------------------------------------------
-void Capture::AddCallstack(CallStack& a_CallStack) {
+void Capture::AddCallstack(HashedCallstack& a_CallStack) {
```

Comment:
```suggestion
void Capture::AddCallstack(HashedCallstack& callstack) {
```

Reply:
I think these classes require some additional polishing. I reverted most of the changes for now as those changes are not necessary for this task. So would you mind if I do it in the separate PR?